lightwood/api/json_ai.py

```diff
@@ -534,6 +534,19 @@ def code_from_json_ai(json_ai: JsonAI) -> str:
 """
     predict_body = align(predict_body, 2)
 
+    predict_proba_body = f"""
+self.mode = 'predict'
+log.info('Cleaning the data')
+data = {call(json_ai.cleaner, json_ai)}
+
+{ts_transform_code}
+
+encoded_ds = lightwood.encode(self.encoders, data, self.target)
+df = self.ensemble(encoded_ds, predict_proba=True)
+return df
+"""
+    predict_proba_body = align(predict_proba_body, 2)
```

Comment:
maybe share this code between `predict` and `predict_proba`

Reply:
As in the code generation bit should be the same except for the last line that does the actual call